### DISCUSSION TITLE
Masque not Displaying Correctly with ElvUI Action Bars

### DISCUSSION THREAD

**Discussion by olvega** [SFX-WoW/Masque#207] [Issues] [answered]
Since the last patch masque is not working optimal with elv ui.
It behaves like it would display half of your bars, 
and then if you /reload ui it will display a  only a couple of bars, and so on, like its living its own life.

elv ui is updated, and i tried to delete masque and its skins, and re installing but the issue is still.
Issue is not there when disabling Masque.

![1d](https://user-images.githubusercontent.com/58878953/70847928-add4ea00-1e6a-11ea-9e9e-a34d23c05eaa.png)
/reload ui
![1g](https://user-images.githubusercontent.com/58878953/70847932-c1805080-1e6a-11ea-8c1d-88e0b7eee57f.png)'

Masque Disabled
![1r](https://user-images.githubusercontent.com/58878953/70847948-05735580-1e6b-11ea-990f-7e8e03f4bdde.png)

**Comment by StormFX** [comment]
I think this is an issue with the latest version(s) of ElvUI. To be sure, type the following in your chat window:

`/luaerror on`

And report any errors you get.

**Comment by StormFX** [comment, marked as answer]
This issue is related to https://www.tukui.org/forum/viewtopic.php?f=40&t=9602. Closing.
